tests/libpod/render.go

```diff
@@ -20,19 +20,30 @@
 package libpod
 
 import (
+	"context"
 	"fmt"
 
+	"github.com/onsi/gomega"
+
 	v1 "k8s.io/api/core/v1"
 	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	v13 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/kubecli"
 
 	"kubevirt.io/kubevirt/pkg/pointer"
-
+	"kubevirt.io/kubevirt/tests/exec"
 	"kubevirt.io/kubevirt/tests/flags"
 	"kubevirt.io/kubevirt/tests/testsuite"
 )
 
+type BlackHoleAction string
+
+const (
+	BlackHoleActionAdd BlackHoleAction = "add"
+	BlackHoleActionDel BlackHoleAction = "del"
+)
+
```

Comment:
Make all of this private resp. hide it? Or better get rid of it alltogether.

Reply:
Done.